dts/bindings/mtd/fujitsu,mb85rc.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Fujitsu MB85RC series I2C FRAM
+
+compatible: "fujitsu,mb85rc"
```

Comment:
Should this include placeholder x's? As in `fujitsu,mb85rcxx`? I couldn't find an existing devicetree binding in the Linux kernel for this.

Reply:
Thanks for your suggestion!